### PR TITLE
doc: thread: add diag module info to cli sample

### PR DESCRIPTION
**Pull request by greg-fer**
Added a note in CLI sample about when to use diag commands.
KRKNWK-6670

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

**Comment by greg-fer** [comment]
@LuDuda , do we want to include a step in Testing for diag commands? Or just mention in a new step when diag commands can be used?

**Comment by LuDuda** [comment]
@greg-fer that would be good. Let me collect some logs from diag.

**Review by MarekPorwisz — APPROVED** [review]

File: samples/openthread/cli/README.rst [hunk 1]
```diff
@@ -85,17 +100,82 @@ After building the sample and programming it to your development kit, test it by
...
+   .. code-block:: console
+
+      uart:~$ ot diag stats
+      received packets: 20
+      sent packets: 0
+      first received packet: rssi=-29, lqi=255
+      last received packet: rssi=-30, lqi=255
+      Done
 
 Dependencies
```
Comment (greg-fer) [inline samples/openthread/cli/README.rst:180]: @LuDuda , @MarekPorwisz , we don't need to update Dependencies?

**Review by LuDuda — APPROVED** [review]
Great work!

**Review by lmaciejonczyk — APPROVED** [review]

File: samples/openthread/cli/README.rst (hunk 1 above)
Comment (LuDuda) [reply samples/openthread/cli/README.rst:180]: There is no special dependencies for Diag module. It just needs to be enabled which it is by default.

**Comment by rlubos** [comment]
Doc build is green, merging.

**Pull request merged**